File: phys/accelerate.js
function acceleration() {
    readline = require("readline-sync")
    var initial_velocity = readline.question("What is the initial velocity?: ")
    var final_velocity = readline.question("What is the final velocity?: ")
    var startiing_time = readline.question("What is the starting line?: ")
    var ending_time = readline.question("What is the ending time?: ")
    var delta_t = ending_time - starting_time
    var delta_v = final_velocity - initial_velocity
}

File: phys/accelerate.py
def acceleration():
    initial_velocity = float(input("What is the initial velocity?: "))
    final_velocity = float(input("What is the final velocity?: "))
    starting_time = float(input("What is the starting time?: "))
    ending_time = float(input("What is the ending time?: "))
    delta_t = ending_time - starting_time
    delta_v = final_velocity - initial_velocity
    acceleration = delta_v/delta_t
    print(f"acceleration = {acceleration} m/s")
acceleration()